churchland_pipeline_python/reference.py

```diff
@@ -46,66 +58,83 @@ def getlocalpath(self):
             elif local_os.lower() == 'dar':
                 path_parts.append('Volumes')
 
+            # append local storage tier name
             path_parts.extend(['Churchland-' + engram_tier, ''])
 
         return os.path.sep.join(path_parts)
+
     
-    def ensureglobal(self, path):
+    def ensureremote(self, path: str) -> str:
+        """Ensures that a path to a storage tier is provided relative to the remote (U19) server."""
+
+        assert len(self)==1, 'Specify one tier'
+
+        return path.replace(self.getlocalpath(), self.getremotepath())
 
-        assert len(self)==1, 'Request one path'
-        return path.replace(self.getlocalpath(), self.getglobalpath())
 
-    def ensurelocal(self, path):
+    def ensurelocal(self, path: str) -> str:
+        """Ensures that a path to a storage tier is provided relative to the local filesystem."""
 
-        assert len(self)==1, 'Request one path'
-        return path.replace(self.getglobalpath(), self.getlocalpath())
+        assert len(self)==1, 'Specify one tier'
 
+        return path.replace(self.getremotepath(), self.getlocalpath())
 
-# Physiology
+
+# ==========
+# PHYSIOLOGY
+# ==========
 
 @schema
-class BrainRegion(dj.Lookup):
+class BrainLandmark(dj.Lookup):
```

Comment:
What's the difference between `BrainLandmark` and `BrainRegion`?

Reply:
In practice, BrainRegion contains areas where we'd actually record neural activity (e.g., primary motor cortex). BrainLandmark contains structures used to identify where those regions actually are (e.g., from MRI scans). Typically these are gyri and sulci, but they could also be structures used to position the head that show up in the scan. I had considered making this an MriLandmark table...it's possible they might be useful for other non-MRI applications though.